components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -397,6 +427,10 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
 
                 String queryString = getQueryString(authenticatorProperties);
                 if (StringUtils.isNotBlank(scopes)) {
+                    if (LoggerUtils.isDiagnosticLogsEnabled() && diagnosticLogBuilder != null) {
+                        diagnosticLogBuilder.inputParam("scopes", scopes)
+                                .inputParam("additional query params", queryString);
```

Comment:
can there be PIIs within these additional query param values?

Reply:
Fixed with 9454f89b9ba4bfe27948b6fa1bbfe6a875db3bb1